src/main/webapp/util/util.js

```diff
@@ -31,6 +31,7 @@ export function setUnifiedConfig(unifiedConfig) {
 }
 
 export function getUnifiedConfigs() {
+    console.log('real file');
```

Comment:
is that console log left here on purpose?

Reply:
oh, no. I am blind

Reply:
I am curious how it passed behind eslint